gax-grpc/src/main/java/com/google/api/gax/grpc/ChannelPool.java

```diff
@@ -231,13 +326,136 @@ private ManagedChannel getNextChannel() {
    *     reverse is not true: Two calls with different affinities might return the same channel.
```

Comment:
I guess the Javadoc "wo calls to this method with the same affinity returns the same channel" is no longer true, since a channel can be removed during refresh?